pandas/tests/io/json/test_normalize.py

```diff
@@ -460,3 +502,39 @@ def test_nonetype_multiple_levels(self):
             'location.country.state.town.info.y': -33.148521423339844,
             'location.country.state.town.info.z': 27.572303771972656}
         assert result == expected
+
+        
+class TestInconsistantlyFormattedToRecord(object):
```

Comment:
Same comment around the use of "inconsistent" - the main take away here is that sometimes the `record_path` will point to an array and other times directly to an object right? `TestRecordPath` might suffice and you can clarify the intent of individual tests in the test name instead

Reply:
Changed to TestMissingKeys